src/constants/commonOptions.js

```diff
@@ -19,6 +19,11 @@ module.exports = {
     placeholder: 'Placeholder',
     tooltip: 'The placeholder text that will appear when this field is empty.'
   },
+  description: {
```

Comment:
And regarding adding the description to the template of all components, I'm overriding the templates myself, but we could add it in the Display tab of all components, if that's what you mean.